core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -282,11 +299,11 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.deleteSkillWithIndex = async function(index) {
-    var elems = await deleteSkillButtons;
+    var elems = await deleteSkillButtons.get(index);
```

Comment:
Can you replace `elems` with a more descriptive name? In this case it's only a single button, so the plural variable name is confusing

Reply:
Done